src/main/java/fi/jubic/quanta/resources/TimeSeriesResource.java

```diff
@@ -20,20 +22,34 @@
 @RolesAllowed({"ADMIN", "USER"})
 public class TimeSeriesResource {
     private final TimeSeriesController timeSeriesController;
+    private final AnomalyController anomalyController;
 
     @Inject
     TimeSeriesResource(
-            TimeSeriesController timeSeriesController
+            TimeSeriesController timeSeriesController,
+            AnomalyController anomalyController
     ) {
         this.timeSeriesController = timeSeriesController;
+        this.anomalyController = anomalyController;
     }
 
     @GET
+    @Path("time-series")
```

Comment:
Query API path cannot be changed.

Reply:
Created `QueryController` to pass the query to correct type of query